pandas/plotting/_core.py

```diff
@@ -21,6 +21,7 @@ def hist_series(
     yrot=None,
     figsize=None,
     bins=10,
+    legend=False,
```

Comment:
could you also add annotation? e.g. `legend: bool = False` , same in other files below

Reply:
I annotated many of the arguments, but there were a few I wasn't sure how to do.

Reply:
thanks, i meant only to add annotation for `legend`, but you've done so many more ^^

> there were a few I wasn't sure how to do
no worries, just leave them aside, it's out of the scope of this PR.